aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -231,5 +231,24 @@ void unary_op(const Tensor& self, const Tensor& output, std::string op_name, Una
                 });
 }
 
+
+TORCH_IMPL_FUNC(remainder_out_mps) (const Tensor& self, const Tensor& output) {
```

Comment:
This function definition is not correct. Infact it should be in BinaryOps.mm. 
Please take a look at the formula to implement:
https://pytorch.org/docs/stable/generated/torch.remainder.html